src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -33,12 +33,16 @@
         c.command('failover', 'failover_replication_link')
         c.command('force-failover', 'failover_replication_link_allow_data_loss')
 
-    # Data Warehouse will not be included in the first batch of GA commands
-    # with s.group('sql db data-warehouse') as c:
-    #     c.command('pause', 'pause_data_warehouse')
-    #     c.command('resume', 'resume_data_warehouse')
+    with s.group('sql dw') as c:
+        c.custom_command('create', 'dw_create')
+        c.command('show', 'get')
```

Comment:
I am quite not sure whether this relates to resource ids or not. 

What this allows would be take a list as input for a parameter where user doesn't need to provide string like `az command --param "a b c"` instead user can say `az command --param a b c` and cli would understand that this parameter accepts list of something. From the description of `--expand` seems like that's what we should have. correct me if i am misunderstanding the param itself.